docs/Microservices/MPS/securityMPS.md

```diff
@@ -15,38 +14,38 @@ In addition to the above assets, there are best practices that are recommended t
+### 5. Web User Credentials
+The Open AMT Cloud Toolkit is designed to operate behind an API Gateway (ex. Kong API Gateway).  The API Gateway's responsibility is to validate the Auth Tokens provided by a user who is requesting access to an API end point.  Once verified the API Gateway will forward the request to the appropriate microservice (MPS or RPS).  In order to make evaluation easy, MPS has implemented an Authorize API end point (see item 3 above) that will issue a JWT when the proper web user credentials are provided.  The Web User credentials are global credentials that are configured in the MPS configuration file and do not provide any fine tuned permissions.  
+*NOTE: This user auth model is very simple and is not recommended for production use.*  
 
 
 ## Best Known Security Methods
 
 ### 1. Enable TLS on network connections
-There are three potential places where TLS should be enabled to protect the security assets:
+There are three potential places where TLS could be enabled to protect the security assets:
 
-* HTTP/WS connection between Web UI and MPS
-* Connection between MPS and Vault
-* Connection between MPS and Intel® AMT device (This is done automatically with default deployments)
+* HTTPS/WSS connection between Web UI and MPS (recommended)
+* Connection between MPS and Vault - If communication between MPS and Vault is outside a secure container environment (not recommended deployment, see item 2 below)
+* Connection between MPS and Intel® AMT device (required and done automatically by MPS)
 
-Encrypting these communication will help prevent network based attacks attempting to control Intel® AMT device. It is recommended that the most modern version of TLS be used to protect these connections.
+Securing these communication routes will help prevent security assets being exposed through network based attacks intercepting messages between components. It is recommended that the most modern version of TLS be used when encrypting communication.
 
 ### 2. Secure and isolate execution environment
-MPS holds the described security assets in memory during execution.  In order to protect these assets while in memory of MPS, it is recommended that MPS be run in a secure execution environment such as a dedicated VM or container. Deploying into a hardened execution environment eases the burden of individually securing the assets while in memory.
+MPS holds several of the described security assets in memory during execution.  In order to protect these assets while in the memory of MPS, it is recommended that MPS be run in a secure execution environment such as a dedicated container. Deploying into a secure conatiner environment eases the burden of individually securing the assets while in memory or in transit between Open AMT Cloud Toolkit services.  Running MPS, RPS, API Gateway, MPS Router, Vault, and Database all within the same secure container instance will help ensure that the communication between these services remains secure.
```

Comment:
MPS holds several security assets in memory during execution.  To protect these assets while in the memory of MPS, run MPS in a secure execution environment such as a dedicated container. Deploying into a secure container environment eases the burden of individually securing the assets while in memory or in transit between Open AMT Cloud Toolkit services.  Running MPS, RPS, API Gateway, MPS Router, Vault, and Database all within the same secure container instance will help ensure that the communication between these services remains secure.